scripts/simplesamlphp/acquia_config.php

```diff
@@ -84,7 +76,14 @@
 
 }
 elseif (getenv('AH_SITE_ENVIRONMENT')) {
-  // Set  ACE ad ACSF sites based on hosting database and site name.
+  /**
+   * Support multi-site and single site installations at different base URLs.
+   *
+   * Overide $config['baseurlpath'] = "https://{yourdomain}/simplesaml/"
+   * to customize the default Acquia configuration.
+   */
+  $config['baseurlpath'] = $protocol . $_SERVER['HTTP_HOST'] . $port . '/simplesaml/';
+  // Set ACE and ACSF sites based on hosting database and site name.
   $config['certdir'] = "/mnt/www/html/{$_ENV['AH_SITE_GROUP']}.{$_ENV['AH_SITE_ENVIRONMENT']}/simplesamlphp/cert/";
   $config['metadatadir'] = "/mnt/www/html/{$_ENV['AH_SITE_GROUP']}.{$_ENV['AH_SITE_ENVIRONMENT']}/simplesamlphp/metadata";
   $config['baseurlpath'] = 'simplesaml/';
```

Comment:
This needs to be removed or it will clobber what you've defined above, right?

Reply:
I'm not sure I understand what you mean. I've just moved around the `$config['baseurlpath'] = $protocol . $_SERVER['HTTP_HOST'] . $port . '/simplesaml/';` line which fixed the missing port 443 in the URL for a multisite we have.

I'm not saying this is best practice or the right fix for BLT, but it's what worked for us. It was really not obvious to understand it has to be within the `elseif (getenv('AH_SITE_ENVIRONMENT'))` block thus why putting it here - assuming it's where it should belong by default - looked best to prevent others from running into the issue.

Reply:
I'm talking about this line, three lines below the code that you inserted: `$config['baseurlpath'] = 'simplesaml/';`

That will clobber the value of baseurlpath that you just assigned.